benchmarks/CMakeLists.txt

```diff
@@ -108,3 +108,7 @@ endfunction()
 add_benchmark(locale_classic src/locale_classic.cpp)
 add_benchmark(random_integer_generation src/random_integer_generation.cpp)
 add_benchmark(std_copy src/std_copy.cpp)
+
+add_benchmark(vector_bool_copy   src/std/containers/sequences/vector.bool/copy/test.cpp)

```

Comment:
No alignment please, since the rest of the file doesn't align.

Reply:
![download](https://user-images.githubusercontent.com/3017120/214367516-40ca9e78-2d53-4f1a-8d75-37a00c525832.jpg)